packages/minuteman/extra/setup_iptables.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+# http://stackoverflow.com/questions/2870992/automatic-exit-from-bash-shell-script-on-error
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+
+RULE="POSTROUTING -m ipvs --ipvs --vdir ORIGINAL --vmethod MASQ -m comment --comment Minuteman-IPVS-IPTables-masquerade-rule -j MASQUERADE"
+
+if ! iptables --wait -t nat -C ${RULE}; then
+  echo "Minuteman IPTables Rule not found: ${RULE}"
+  echo "Inserting Rule"
+  iptables --wait -t nat -I ${RULE} && echo "Rule successfully inserted"
+else
+  echo "Minuteman IPTables rule found"
```

Comment:
How are we verifying / testing that? This adds all this bash code which does that in theory, but we don't actually test that it works like that, esp. since some of the conditions around upgrade are harder to get into.